src/SourceBuild/content/repo-projects/vstest.proj

```diff
@@ -6,6 +6,9 @@
     <BuildCommandArgs>$(BuildCommandArgs) /p:SemanticVersioningV1=true</BuildCommandArgs>
     <BuildCommand>$(ProjectDirectory)\eng\common\build$(ShellExtension) $(BuildCommandArgs)</BuildCommand>
     <DeterministicBuildOptOut>true</DeterministicBuildOptOut>
+
+    <!-- https://github.com/dotnet/source-build/issues/3814 -->
+    <NoWarn>$(NoWarn),IDE0044</NoWarn>
```

Comment:
Curious, where is that NoWarn setting respected when building the repo?

Reply:
Yep, I just saw that myself. Confirmed that the build is not fixed by this. My bad. I will create a patch to address the code fixes. They are pretty simple.